monitoring/views/mlops_flow_run_cost.view.lkml

```diff
@@ -0,0 +1,34 @@
+view: mlops_flow_run_cost {
+  derived_table: {
+    sql: SELECT
+        *
+      FROM `moz-fx-data-shared-prod.monitoring_derived.outerbounds_cost_per_flow_run_v1`
+      ORDER BY invoice_day DESC
+      ;;
+  }
```

Comment:
Similar to above comment

Reply:
Oh! Do we have an example view that does this? I was going off examples in the repo (the videos I found were largely solving for a different use case than this), and I didn't happen to come across that option!

Reply:
Here's an example: https://github.com/mozilla/looker-spoke-default/blob/81f8b74302a2a16c700cef6dfc524cf1739c29b6/firefox_desktop/views/newtab_clients_daily.view.lkml#L3, it doesn't include the `ORDER BY`, but Looker's default sorting does this with date's selected in an Explore/dashboard query IIRC